client/pdo/client/commands/collection.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import logging
+import os
+import toml
+import typing
+
+from zipfile import ZipFile
+
+import pdo.client.builder.shell as pshell
+import pdo.client.builder.script as pscript
+import pdo.client.builder as pbuilder
+from pdo.common.utility import experimental
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    'export_contract_collection',
+    'import_contract_collection',
+    'script_command_export',
+    'script_command_import',
+    'do_collection',
```

Comment:
Aren't these three commands usually used only indirectly using below `load_commands`? If so, why export by default?

Reply:
because there are times when we call into the script functions directly. that's part of the reason for the class interface. this is consistent with other commands.